pkgs/tools/backup/mydumper/default.nix

```
@@ -1,29 +1,57 @@
-{ lib, stdenv, fetchFromGitHub
-, cmake, pkg-config, sphinx
-, glib , pcre
-, libmysqlclient, libressl
-, zlib, zstd
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  sphinx,
+  glib,
+  pcre,
+  libmysqlclient,
+  libressl,
+  zlib,
+  zstd,
+  nix-update-script,
+  testers,
+  versionCheckHook,
+  mydumper,
 }:
 
 stdenv.mkDerivation rec {
   pname = "mydumper";
-  version = "0.14.3-1";
+  version = "0.16.3-6";
 
   src = fetchFromGitHub {
-    owner  = "mydumper";
-    repo = "mydumper";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-qyJGnrBOElQ3s2VoOWfW1luacd33haanmzKidMBgCpc=";
+    owner = "
+    mydumper ";
+    repo = "
+    mydumper ";
```

looks like these two lines got unintentionally changed

```suggestion
    owner = "mydumper";
    repo = "mydumper";
```

---

whoops, thanks